src/content/docs/en/guides/cms/decap-cms.mdx

```diff
@@ -9,12 +9,28 @@ i18nReady: true
 import Grid from '~/components/FluidGrid.astro'
 import Card from '~/components/ShowcaseCard.astro'
 
-[Decap CMS](https://www.decapcms.org/) (formerly Netlify CMS) is an open-source, Git-based content management system.
+[Decap CMS](https://www.decapcms.org/) (formerly Netlify CMS) is an open-source, Git-based content management system. When using Decap you will be able to use Astro's [Content Collections](/en/guides/content-collections/) as well as [Image Optimization](/en/guides/images/).
+
+Decap adds a route (typically `/admin`) to your project that will load a React app which allows authorized users to manage their content directly from the deployed website. Whenever a user makes changes to their content, Decap will process those changes through git, which typically will trigger continuous deployment.
+
+## Integrating with Astro
+
+There are two options for using Decap with Astro:
+
+1. Loading the default package, either via [unpkg](https://www.unpkg.com) or importing the package into a `<script>` tag.
+2. [Manually initializing Decap](https://decapcms.org/docs/manual-initialization/).
+
+### Common practices
+
+Many users will add their admin route and config file into their project's `public` folder. However it is equally viable to have this route be a `.astro` file in the [Pages](/en/basics/astro-pages/) directory.
+
+Decap CMS was originally developed by Netlify so it has first class support for [Netlify Identity](https://docs.netlify.com/security/secure-access-to-sites/identity/). When deploying to hosting providers other than Netlify, it is necessary to create your own OAuth routes. These routes are often created as standalone serverless functions for static sites, but it is possible to use Astro's On-demand Rendering to create these routes within your Astro Project.  See [Decap's OAuth Docs](https://decapcms.org/docs/external-oauth-clients/) for more details.
```

Comment:
This is actually one of my main reasons for wanting to update this doc. Netlify has abandoned this project all together and it looks like their support for netlify identity is dropping in quality (saw complaint in decap discord)

Decap is not at all tied to netlify and I want to make that more clear

Reply:
OK, so then it's NOT a happy path! Gotcha!

In that case, instead of a "Common Practices" thing, then how about sections that literally are for those things.

```suggestion
## Adding Authentication

### Decap CMS with Netlify Identity

Decap CMS was originally developed by Netlify and has first class support for [Netlify Identity](https://docs.netlify.com/security/secure-access-to-sites/identity/). This allows you to have JSON web token (JWT)-based user authentication for your sites deployed to Netlify. You can use this for gated content, site administration, and more.

### Decap CMS with External OAuth Clients

When deploying to hosting providers other than Netlify, you must create your own OAuth routes.

In Astro, this can be done with on-demand rendered routes in your project configured with [`server` or `hybrid` output](/en/guides/server-side-rendering/#enable-on-demand-server-rendering) enabled.

See [Decap's OAuth Docs](https://decapcms.org/docs/external-oauth-clients/) for a list of compatible community-maintained OAuth clients.
```